docs/explanation/landscape/landscape-client-architecture.md

```diff
@@ -0,0 +1,76 @@
+(explanation-client-architecture)=
+# Landscape Client architecture
+
+Landscape Client is the client-side component of the Landscape ecosystem.
+It is made up of a collection of processes.
+This document explains these processes and their purposes.
+
+A Landscape Client has four main services:
+
+  * [Watchdog](#watchdog) — initializes and supervises other long-running processes
+  * [Broker](#broker) — exchanges messages with Landscape Server
+  * [Monitor](#monitor) — collects information from the client
+  * [Manager](#manager) — performs actions and applies side effects on the client
+
+There are also important processes that run either periodically or on demand:
+
+  * [Package reporter](#package-reporter) - reports the status of packages on the client
+  * [Package changer](#package-changer) - applies package changes
+  * [Config](#config) - configures the client
+  * [Release upgrade manager](#release-upgrade-manager) — upgrades the operating system to a new Ubuntu release
```

Comment:
Couple things:

- Can you change all the dashes to be consistent? you can just use the - hyphen
- If there isn't a reason for these to be non-alphabetical, can you make them alphabetical in their sections? (and also alphabetize them below - not in the section order, but alphabetized across all of them)